index.html

```diff
@@ -533,6 +584,63 @@
 
             </div>
         </div>
+
+        <div id="settings-modal" class="modal fade">
+            <div class="modal-dialog modal-lg">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                        <h4 class="modal-title">Settings</h4>
+                    </div>
+                    <div class="modal-body">
+                        <div class="split-container-left">
+                          <p>
+                            <b>
+                              Board and Pieces
+                            </b>
+                            <br/>
+                            <i>Changes to board and pieces will only take effect, once you watch or start a new game.</i><br/>
```

Comment:
Going to go with that. If we do this however, we can do the same thing to the wall-direction. That way both options can be changed dynamically.